src/libraries/System.Formats.Tar/tests/TarFile/TarFile.CreateFromDirectory.File.Roundtrip.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using Xunit;
+
+namespace System.Formats.Tar.Tests
+{
+    public class TarFile_CreateFromDirectory_Roundtrip_Tests : TarTestsBase
+    {
+        [ConditionalFact(typeof(MountHelper), nameof(MountHelper.CanCreateSymbolicLinks))]
+        public void WriteSymlinkTargetsInsideTheArchiveRoundtrips()
```

Comment:
Could you please extend the test to have a variety of link target paths and the link itself being in multiple levels (i.e: at 0 (root), and at 1).
e.g:
```
link-> ./file.txt // pass.
link-> ../file.txt // should throw TarExtractingResultsLinkOutside
subDir1/link-> ./file.txt // pass.
subDir/link-> ../file.txt // pass.
subDir/link-> ../../file.txt // should throw TarExtractingResultsLinkOutside
```